tests/Application/config/bundles.php

```diff
@@ -67,4 +67,8 @@
     $bundles[Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle::class] = ['all' => true];
 }
 
+if (SyliusKernel::MINOR_VERSION == 13) {
```

Comment:
```suggestion
if (Sylius\Bundle\CoreBundle\SyliusCoreBundle::VERSION_ID >= 11300) {
```
And I believe you can remove the above if-else.